markdown/usage/data_guidelines.md

```diff
@@ -0,0 +1,48 @@
+
+Archiving is often the responsibility of your host institution, contact them for more details.
+
+## Data sharing
+
+In the era of [FAIR](https://www.force11.org/group/fairgroup/fairprinciples) (Findable, Accessible, Interoperable and Reusable) and [Open science](https://www.vr.se/english/mandates/open-science/open-access-to-research-data.html), datasets should be made available to the public, for example by submitting your data to a public repository.
+
+### Choosing a repository
+
+It’s recommended to choose a domain-specific repository when possible. It is also important to consider the sustainability of the repository to ensure that the data will remain public. Please see[SciLifeLab's data guidelines](https://scilifelab-data-guidelines.readthedocs.io/en/latest/docs/index.html) or the [EBI archive wizard](https://www.ebi.ac.uk/submission/) for suggestions depending on data type. You can also refer to the [ELIXIR Deposition Databases](https://elixir-europe.org/services/tag/elixir-deposition-databases) and [Scientific Data’s Recommended Data Repositories](https://www.nature.com/sdata/policies/repositories), to find suitable repositories.
+
+For datasets that do not fit into domain-specific repositories, you can use an institutional repository when available (e.g. [SciLifeLab Data Repository](https://scilifelab.figshare.com/)) or a general repository such as [Figshare](https://figshare.com/) and [Zenodo](https://zenodo.org/).
+
+### Preparing for submission
+
+#### **Describing and organizing your data**
+
+Metadata should be provided to help others discover, identify and interpret the data. Researchers are strongly encouraged to use community metadata standards and ontologies where these are in place, consult e.g [FAIRsharing.org](https://fairsharing.org/databases/). Data repositories may also provide guidance about metadata standards and requirements. Capture any additional documentation needed to enable reuse of the data in Readme text files and [Data Dictionaries](https://help.osf.io/hc/en-us/articles/360019739054-How-to-Make-a-Data-Dictionary) that describe what all the variable names and values in your data really mean. Identifiers to refer to e.g. ontology terms can be designed for computers or for people; in a FAIR data context it is recommended to supply both a human-readable as well as a machine-resolvable Persistent Identifier (PID) for each concept used in the data.
+
+#### **Data integrity**
+
+Some repositories require md5 checksums to be uploaded along with the files. Also consider adding checks that your data files follow the intended file formats and can be opened by standard software for those formats.
+
+Consult the docs/output page to see if the pipeline generates the sums automatically and where to find them.
+
+/ Feedback from nf-core team needed /: Or where should this be specified?
```

Comment:
```suggestion
If you are using a Linux system, you can generate md5 checksums using the `md5sum` command.
```

I think just omit this for now. Some pipelines already generate md5 sums, some do not. It's perhaps a good idea to make this a guideline for all nf-core pipelines to try to expand this support out, but that should be a separate issue.